src/uwtools/tests/test_rocoto.py

```diff
@@ -119,10 +119,12 @@ def test__add_compound_time_string_basic(self, config, instance, root):
         assert child.text == str(config)
 
     def test__add_compound_time_string_cyclestr(self, instance, root):
-        config = {"cyclestr": {"attrs": {"baz": "42"}, "value": "qux"}}
+        config = {"cyclestr": {"attrs": {"offset": "00:05:00"}, "value": "qux"}}
```

Comment:
Many of the changes you see to the config dicts in this file are to get them to pass the validation step and/or to be more realistic if others peek in here for examples.